unlock_signature.go

```diff
@@ -17,3 +17,7 @@ func (s *SignatureUnlock) Type() UnlockType {
 func (s *SignatureUnlock) Size() int {
 	return util.NumByteLen(byte(UnlockSignature)) + s.Signature.Size()
 }
+
+func (s *SignatureUnlock) WorkScore(workScoreStructure *WorkScoreStructure) WorkScore {
+	return workScoreStructure.WorkScoreEd25519Signature
```

Comment:
Btw, we never charge for these right? Because we never analyse inputs anyway? Maybe we should switch to charge costs if someone creates these things on the output side.

Reply:
We do charge for these in when calculating the workscore on the transaction essence.

Reply:
Added the Signature.WorkScore() func